reconciletags/config_integrity_test.py

```diff
@@ -0,0 +1,59 @@
+"""Tests to check the integrity of json config files.
+
+These tests assume that the json configs live in a top-level
+folder named config."""
+
+import glob
+import json
+import logging
+import subprocess
+import unittest
+
+
+class ReconcilePresubmitTest(unittest.TestCase):
+
+    # This function requires gcloud be installed and authenticated
+    # with your project
+    def _get_digests(self, repo):
+        try:
+            output = json.loads(
+                subprocess.check_output(['gcloud', 'beta', 'container',
+                                         'images', 'list-tags',
+                                         '--no-show-occurrences',
+                                         '--format=json', repo]))
+            # grab the digest for each image and strip off the 'sha256:'
+            # for matching purposes
+            digests = [image['digest'].split(':')[1] for image in output]
+            return digests
+        except:
```

Comment:
could you narrow this try/except to only catch a specific error, and also log the raised error? This could also throw if the json.loads fails, for instance.